babel-worker.cjs

```diff
@@ -0,0 +1,39 @@
+const { transformSync } = require("@babel/core");
```

Comment:
We deviates from the conventional Gulp stream approach because: 

1) Passing the source code to sub process is more expensive than passing the path
2) Passing the transformed code to main process in order to written via `gulp.dest` is expensive and ultimately counters the performance gain from multiple process

So we end up creating our own gulp-babel + gulp-newer in this file.